Resources/config/controller.xml

```diff
@@ -19,7 +19,7 @@
             <argument>%assetic.variables%</argument>
         </service>
         <service id="assetic.controller" class="%assetic.controller.class%" scope="prototype">
-            <argument type="service" id="request" />
+            <argument type="expression">service('request_stack').getMasterRequest()</argument>
```

Comment:
It would be much better to actually use the `request_stack` service here. The usage of an expression was done temporarily as a hack in Symfony and will be removed soon when the new asset component will land into 2.7.